### PR TITLE
Switch to std-uritemplate

### LINKED ISSUE #77: swap tavis uri template for std uri template
@andreaTP as soon as you think you're ready, please go ahead and submit a PR here.

### PR DESCRIPTION
**Pull request by andreaTP**
Fix #77

### CODE REVIEW

File: tests/RequestInformationTest.php [hunk 1]
```diff
@@ -93,16 +93,25 @@ public function testPathParametersOfDateTimeOffsetType(): void
...
+    }
+
+    public function testCanHandleNumericTypes(): void {
         // Arrange as the request builders would
         $requestInfo = new RequestInformation();
         $requestInfo->httpMethod = HttpMethod::GET;
         $requestInfo->urlTemplate = "http://localhost/users{?%24count}";
 
-        $requestInfo->setPathParameters(['%24count' => true]);
+        $requestInfo->setPathParameters(['%24count' => 1]);
```
Comment (andreaTP) [inline tests/RequestInformationTest.php:114]: the previous implementation used boolean as numbers, which is arguably not correct.

**Comment by andreaTP** [comment]
@Ndiritu are you waiting for me to do anything to unblock this one?

**Comment by andreaTP** [comment]
Rebased on latest `dev`, kind ping @Ndiritu and @baywet

**Review by baywet — APPROVED** [review]
Thanks for the nudge @andreaTP . @Ndiritu for final review and merge. (the team in Nairobi is having a set of events this week, expect things to be slower).
Also I believe the version needs to be bumped and a changelog entry needs to be added.

**Comment by andreaTP** [comment]
I missed reacting to the feedback, sorry for the delay ...

> Also I believe the version needs to be bumped and a changelog entry needs to be added.

There is no CHANGELOG in this repo and the versions are directly coming from git as far as I understand it.

**Comment by baywet** [comment]
@andreaTP https://github.com/microsoft/kiota-abstractions-php/blob/b69f6cfeafa46ce1df092971602f5dec677fa39a/src/Constants.php#L7

@Ndiritu can you clarify how a version gets released please? do you manually create the tag and release on the repo today?

**Comment by SilasKenneth** [comment]
Hi @baywet ,

This is currently manually written. 
There is plan to automate the process as we do in the service libraries.

**Review by SilasKenneth — APPROVED** [review]

**Review by baywet — APPROVED** [review]
Ok thanks for the additional context @SilasKenneth 
Can you please merge and release this one?

**Pull request merged**